all_fruits.py

```diff
@@ -0,0 +1,163 @@
+import random, pygame, sys, math
+from pygame.locals import *
+
+white = (255, 255, 255)
+green = (0, 255, 0)
+
+class Fruit(pygame.sprite.Sprite):
+    def __init__(self, screen, image_name):
+        # Call the parent class (Sprite) constructor
+        pygame.sprite.Sprite.__init__(self)
+        self.speed = .3
+        self.screen = screen
+        self.total_time = 0
+        self.time = 0
+
+        self.image =  pygame.image.load(image_name).convert_alpha()
+        self.image.set_colorkey(white)
+        self.image.convert_alpha()
+        self.rect = self.image.get_rect()
+
+        self.x = random.randint(0, self.screen.get_width()-int(self.image.get_width()/2))
+        self.y = self.screen.get_height()+self.image.get_width()
+
+        self.rect.centerx = self.x + (self.image.get_width()/2)
+        self.rect.centery = self.y + (self.image.get_height()/2)
+
+        #figure out direction to fruit to move based on spawn position (left or right)
+        border = screen.get_width()/2
```

Comment:
The rest of this init function could almost be it's own method that init calls. It's determining how the fruit will move so it's can be sectioned off in a comprehensible manner and there are only two class attributes that are created in 15 lines of code.